src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/commands.py

```diff
@@ -130,3 +135,11 @@ def load_command_table(self, _):
         g.command('list', 'list_authorization_rules')
         g.show_command('show', 'get_authorization_rule')
         g.command('keys list', 'list_keys')
+
+# MigrationConfigs Region
+    with self.command_group('servicebus migration', sb_migration_util, client_factory=migration_mgmt_client_factory) as g:
+        g.command('start', 'create_and_start_migration')
```

Comment:
IMO the migration process would benefit from using custom commands instead of purely reflecting on the SDK.  This approach requires some kind of guide to really understand the process and sequence the necessary calls.  Ideally, I could do just the following:
`az servicebus migration start …` and it would start the migration, poll until it was done, the complete the process (switch the connection strings) and clean up (delete) the migration object.  From our meeting, that's probably too much than you'd like, but you could also reduce this to two commands:

```
az servicebus migration start …  # start migration and poll until done
az servicebus migration complete --clean-up  # complete migration and clean up the migration object
```
This would be a more convenient 2-step solution to the current process:
```
az servicebus migration start
az servicebus migration show # as many times as needed until done
az servicebus migration complete
az servicebus migration delete
```

For this, you would make "start" and complete custom commands.

```Python
def start_servicebus_migration(cmd, client, …, no_wait=False):
  if no_wait:
    # This is effectively what the current command would do...
    return client.create_and_start_migration(…)

  client.create_and_start_migration(…)

  # pseudocode--obviously don't poll infinitely and handle exceptions :)
  while(True):
    time.sleep(30) # some delay
    result = client.get(…)
    if result.code == 200:
      # this lets the user know what to do next. Atypical, but this is a one-time
      # interactive command so this kind of this is acceptable
      logger.warning('Migration to Premium namespace is finished. Run `az servicebus migration complete` to switch connection strings and complete the migration process.'
      return
```

Then complete just looks like:
```Python
def complete_migration(cmd, client, …, clean_up=False):
  client.complete_migration(…)

  # good reminder for what connection string references which namespace
  # helps clarify the confusing "post-migration-name" parameter.
  logger.warning('Servicebus namespace migration complete. Use the following connection strings to access your namespaces:)
  logger.warning('\tStandard: %s', standard_cs)
  logger.warning('\tPremium: %s', premium_cs)
  if clean_up:
    client.delete(…)
  return 
```

Reply:
we have changed the start command to wait until all entity replication is completed, now the user will not have to pool upon the 'PendingEntitycount' property using show command.

After completing migration, the existing connection strings to ‘standard’ namespace will connect to ‘premium’ namespace automatically.